_includes/templates/iot-gateway/remote-creating-connector-mqtt.md

```diff
@@ -16,121 +16,247 @@ After running docker image, you can see the following logs in your terminal:
+It will include a data mapping for the **“data/”** topic, specifying a JSON converter and defining attribute and timeseries mappings for device data. 
+Additionally, it will handle connect and disconnect requests for sensors with expressions to extract device names from topic filters.
 
 To create a connector, follow these steps:
 
 {% assign addNewConnector = '
     ===
         image: /images/gateway/dashboard/gateway-getting-started-7-ce.png,
-        title: Click "**Connectors configuration**" button on the right panel.
+        title: Click on "**Connectors configuration**" button on the right panel;
     ===
-        image: /images/gateway/dashboard/gateway-getting-started-mqtt-8-ce.png,
-        title: Click the "**+**" button, fill in "**Name**", "**Type**" and "**Logging level**" fields, paste your connector configuration into **Configuration** field and click on **Save** button.
+        image: /images/gateway/dashboard/gateway-getting-started-8-ce.png,
+        title: Click the "**+ Add connector**" button;
     ===
         image: /images/gateway/dashboard/gateway-getting-started-mqtt-9-ce.png,
-        title: Connector has been successfully added.
+        title: Choose "**MQTT**" connector type from the dropdown, fill in "**Name**" field, choose "**Logging level**" to INFO, disable "**Fill configuration with default values**" field and click on "**Add**" button;
     ===
         image: /images/gateway/dashboard/gateway-getting-started-mqtt-10-ce.png,
-        title: Toggle the switch to enable the connector.
+        title: Connector created.
 '
 %}
 
 {% include images-gallery.liquid showListImageTitles="true" imageCollection=addNewConnector %} 
 
-Following the steps outlined, your gateway will receive and apply the new configuration. It will then synchronize 
-its state with the remote server. You can view the synchronization status of the connector configuration 
-in the "**Configuration**" column, which will indicate whether the gateway is successfully aligned with 
-the remote settings.
+First, we need to configure the connection to the demo broker. Let’s start from the "**Connection to broker**" section. 
+This section offers detailed connection configuration options and contains several important fields, including host, port, MQTT version, client ID, and security settings. 
+The **host** field specifies the address of the broker, while the **port** field indicates the communication port. The **MQTT version** field ensures compatibility with the protocol version being used. The **client ID** uniquely identifies the client, and the **security** settings provides configuration for client authorization at MQTT Broker.
+
+Fill in the following fields with values:
+
+| **Field name** | **Value**            |
+|:---------------|:---------------------|
+| Host           | host.docker.internal |
+| Port           | 1884                 |
+| MQTT version   | 5                    |
+| Client ID      | randomly generated   |
```

Comment:
Probably it is not necessary to write about randomly generated here, because it is more like description for the field and it should be generated automatically on UI. So, it is not required to change it manually.